docker-compose.link.yaml

```diff
@@ -13,7 +13,16 @@ services:
         npm run start:watch"
     networks:
       - ffc-demo
+    ports:
+      - "3003:3003"
+
+  ffc-demo-claim-postgres:
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
```

Comment:
Volume declaration duplicated from `docker-compose.override.yaml` in order preserve data for running service without creating an equivalent volume when running tests.